.github/workflows/build-test.yml

```diff
@@ -21,15 +21,16 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Calculate file differences
-        uses: lots0logs/gh-action-get-changed-files@2.1.4
-        id: diff
-        with:
-          token: ${{ secrets.GITHUB_TOKEN }}
-
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Calculate file differences
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          json: true
+          json_raw_format: true
+
```

Comment:
Can we keep minimized changes here?

Reply:
> not moving the location

No. `tj-actions/changed-files` must have `actions/checkout` executed beforehand.
ref: https://github.com/odan-sandbox/changed-files-sandbox/actions/runs/4513265067/jobs/7947867418

Reply:
Got it, thanks!